src/openrct2/network/Socket.cpp

```diff
@@ -245,6 +255,45 @@ class Socket
             return true;
         }
     }
+    
+    /*
+     * https://oliver-kaestner.de/english-c-query-srv-dns-record-with-example/
+     */
+    static bool ResolveSRV(const char* host, SRVRecord* resolved) {
+        struct __res_state res;
```

Comment:
I guess we would only need the `struct` if this was C code.
```suggestion
        __res_state res;
```